test/common/router/route_fuzz_test.cc

```diff
@@ -16,6 +18,19 @@ namespace {
 // Limit the size of the input.
 static const size_t MaxInputSize = 64 * 1024;
 
+bool isUnsupportedRouteConfig(const envoy::config::route::v3::Route& route,
+                              bool extended_check = false) {
+  if (route.has_filter_action() || route.has_non_forwarding_action()) {
+    return true;
+  }
+  if (extended_check && route.has_route() &&
+      !(route.route().has_cluster_specifier_plugin() || route.route().has_cluster() ||
+        route.route().has_cluster_header() || route.route().has_weighted_clusters())) {
+    return true;
+  }
```

Comment:
I think these 4 lines are not needed.

Reply:
With the corpi present, these lines are indeed not needed. Removed.